packages/grafana-toolkit/README.md

```diff
@@ -1,55 +1,82 @@
-# Grafana Toolkit
 
-Make sure to run `yarn install` before trying anything!  Otherwise you may see unknown command grafana-toolkit and spend a while tracking that down.
+# grafana-toolkikt
+grafana-toolkit is a simple CLI that enables efficient Grafana extensions development
```

Comment:
It's generally best to avoid words like "easy" or "simple", since we typically don't know what experience the developer has. It's not a good feeling to struggle with something that is regarded "simple".